packages/api/models/Error.ts

```diff
@@ -9,5 +9,5 @@ export interface Error {
 }
 
 export interface ValidationError extends Error {
-  errorMap: Record<string, any>;
+  errorMap: Record<string, never>;
```

Comment:
```suggestion
  errorMap: Record<string, never>;
```

I wasn't entirely convinced that `never` was appropriate here. I don't disagree with it's use but can we talk through why?

Reply:
That was me taking the first IDE suggestion and then subsequently getting an education into the three options there. Looks like `unknown` is more appropriate, if not just making it `string` outright. And actually, I believe we settled on an in-line type approach when we discussed it with @tonysumner-ccpo. E.g. 

Input Form:
```
{
  "pf_name": "this is too long",
  "description": "a description",
  "apps": [
    {
      "app_name": "app1",
      "price": 100
    },
    {
      "app_name": "app2",
      "price": 1000000
    }
  ]
}
```

HTTP 400 / Validation Error Response Body:
```
{
  "pf_name": {
      "input": "this is too long",
      "message": "Must be fewer than 250 characters."
    },
  "description": "a description",
  "apps": [
    {
      "app_name": "app1",
      "price": 100
    },
    {
      "app_name": "app2",
      "price": {
        "input": 1000000,
        "message": "Must be less than 1000."
    }
  ]
}
```

Note that this is totally notional and we can certainly tweak it to make life easier on both the function and UI side. But I am mentioning here mostly just to say that it probably doesn't matter what we settle on in this particular PR since I think it'll end up getting blown away. :)